# 7.2 Hypothesis Testing for the Population Variance

## Variance and the Test Statistic

In the previous chapter, we learned how we test a claim about the mean or the proportion of a population. The process for testing a claim about the variance of a population is very similar. 

(Recall that the **variance** is simply the square of the standard deviation; that is, if $\sigma$ is the population standard deviation, then $\sigma^2$ is the population variance. While the variance and the standard deviation are not quite the same, the two parameters both measure how far data is from the mean. Testing the population variance is the same as testing the population standard deviation.)

Unlike our tests for the population mean or the population proportion, our test for the population variance uses a test statistic with a $\chi^2$-distribution. The test statistic is

$$ \chi^2 = \frac{(n - 1)s^2}{\sigma^2}, $$

where $n$ is the sample size, $s$ is the sample standard deviation, and $\sigma$ is the hypothesized population standard deviation. The degrees of freedom of the $\chi^2$-distribution of point estimates is one less than the sample size: $df = n - 1$. The population must be normally distributed for the test to be valid.

The general steps for completing a hypothesis test have not changed:
1. State the null and alternative hypotheses.
2. Assuming the null hypothesis is true, determine the features of the distribution of test statistics.
3. Find the $p$-value of the test statistic.
4. Make a conclusion about the null hypothesis.

***

### Example 2.1
A beverage company wants to make sure that the amount of soda in their two-liter bottles doesn't differ significantly bottle-to-bottle. Specifically, they want the population standard deviation to be no larger than 1.5 milliliters. The quality control division of the company carefully measures the amount of soda in 13 bottles and obtains the following results (in milliliters):
<br/>
<br/>

<center>2002.08, 1999.25, 2002, 1996.51, 1998.99, 2000.06, 1999.73, 2001.16, 2001.19, 2002.14, 2000.21, 2002.15, 2000.06</center>

Conduct a hypothesis test at the 10% significance level to determine is the beverage company's criterion is satisfied.

#### Solution
##### Step 1: State the null and alternative hypotheses.
The beverage company doesn't want the population standard deviation to be larger than 1.5 milliliters; that is, the company wants $\sigma \leq 1.5$. Since the symbol has an 'equal' in it, this is the null hypothesis. The alternative hypothesis is the opposite of the null hypothesis.

$$\begin{align}
H_0:&\ \sigma \leq 1.5 \\
H_a:&\ \sigma > 1.5
\end{align}$$

Note, we have chosen to write our hypotheses in terms of the population standard deviation $\sigma$. But we could also have written the hypotheses in terms of the population variance $\sigma^2$. Since $1.5^2 = 2.25$, the same hypotheses written in terms of the population variance are

$$\begin{align}
H_0:&\ \sigma^2 \leq 2.25 \\
H_a:&\ \sigma^2 > 2.25
\end{align}$$

These two ways of writing the hypotheses are equivalent and mean the same thing.

Also, recall that since the alternative hypothesis uses a greater-than symbol, we will be performing a right-tailed test.

##### Step 2: Assuming the null hypothesis is true, determine the features of the distribution of test statistics.
Because we are testing the population standard deviation, we will use a $\chi^2$ distribution with degrees of freedom

$$ df = n - 1 = 13 - 1 = 12. $$

##### Step 3: Find the $p$-value of the test statistic.
To calculate the test-statistic, we first need to find the sample standard deviation. To begin, we calculate the sample mean:

In [1]:
x = c(2002.08, 1999.25, 2002, 1996.51, 1998.99, 2000.06, 1999.73, 2001.16, 2001.19, 2002.14, 2000.21, 2002.15, 2000.06)
n = length(x)

xbar = sum(x)/n
xbar

[1] 2000.425

The sample mean is about $\bar{x} = 2000.4254$. Next, we calculate the sample standard deviation:

In [2]:
s = sqrt(sum( (x - xbar)^2 )/(n - 1))
s

[1] 1.628515

The sample standard deviation is $s = 1.6285$. Assuming the null hypothesis is true (so that $\sigma = 1.5$), we calculate the test statistic
$$ \chi^2 = \frac{(n - 1)s^2}{\sigma^2}. $$

In [3]:
sigma = 1.5
chisq = (n - 1)*s^2 / sigma^2
chisq

[1] 14.14432

Our test statistic is $\chi^2 = 14.1443$. Then since we are performing a right-tailed test, the $p$-value is equal to $P(\chi^2 > 14.1443)$. We use R to calculate this probability.

In [4]:
pvalue = pchisq(q = 14.1443, df = 12, lower.tail = FALSE)
pvalue

[1] 0.2915886

So the $p$-value is $P(\chi^2 > 14.1443) = 0.2916$. That means that, assuming the null hypothesis is true, there is a 29.16% chance that a random sample of 13 bottles would have a sample standard deviation as extreme as ours.

##### Step 4: Make a conclusion about the null hypothesis.
The level of significance is 10%, so $\alpha = 0.10$. But since

$$p\text{-value} = 0.2916 \geq 0.10 = \alpha, $$

we do not reject the null hypothesis.

There is not enough evidence to conclude that the population standard deviation of the bottles of soda is greater than 1.5 milliliters.

***

### Example 2.2
A medical testing company has created an IQ test and claims that the population scores an average of 100 points on the test with a standard deviation of 15 points. A psychologist suspects that the standard deviation might be wrong. He administers the IQ test to 18 randomly chosen individuals and obtains the following scores.
<br>
<br>

<center>
    108, 95, 98, 101, 118, 105, 102, 80, 112, 113, 88, 101, 93, 98, 104, 99, 87, 90
</center>

Perform a hypothesis test with a 5% level of significance to test the medical testing company's claim.

#### Solution
##### Step 1: State the null and alternative hypotheses.
The medical testing company claims that the population standard deviation score on the test is 15 points, or $\sigma = 15$. Since the symbol is an equal sign, this is our null hypothesis. The alternative hypothesis is the opposite of this.

$$\begin{align}
H_0:&\ \sigma = 15 \\
H_a:&\ \sigma \neq 15
\end{align}$$

Since the alternative hypothesis uses a not-equal-to symbol, we will be performing a two-tailed test.

##### Step 2: Assuming the null hypothesis is true, determine the features of the distribution of test statistics.
Since we are testing the population standard deviation, we will be using a $chi^2$-distribution with $n - 1 = 18 - 1 = 17$ degrees of freedom.

##### Step 3: Find the $p$-value of the test statistic.
To calculate the point estimate, we first must find the sample standard deviation. And to find the sample standard deviation, we need the sample mean.

In [1]:
x = c(108, 95, 98, 101, 118, 105, 102, 80, 112, 113, 88, 101, 93, 98, 104, 99, 87, 90)
n = length(x)

xbar = sum(x)/n
xbar

[1] 99.55556

So the sample mean is $\bar{x} = 99.5556$. Next, we use our calculation of the sample mean to find the sample standard deviation.

In [2]:
s = sqrt( sum( (x - xbar)^2 )/(n - 1) )
s

[1] 9.823275

The sample standard deviation is $s = 9.8233$. Using this with the hypothesized population standard deviation of $\sigma = 15$, we calculate the test statistic using the formula

$$ \chi^2 = \frac{(n - 1)s^2}{\sigma^2}. $$

In [3]:
sigma = 15
chisq = (n - 1)*s^2 / sigma^2
chisq

[1] 7.290864

So the sample statistic is $\chi^2 = 7.2909$. Since we are performing a two-tailed test, and because our sample standard deviation $s = 9.8233$ is smaller than our hypothesized population standard deviation of $\sigma = 15$, we will find the half of the $p$-value in the lower tail by calculating $P(\chi^2 < 7.2909)$.

In [4]:
half_pvalue = pchisq(7.2909, df = 17, lower.tail = TRUE)
half_pvalue

[1] 0.02054031

To find the full $p$-value, we double this quantity.

In [5]:
pvalue = 2*half_pvalue
pvalue

[1] 0.04108062

So the probability of testing 18 randomly sampled individuals from the population and obtaining a sample standard deviation as extreme as the one we got is only about 4.11%.

##### Step 4: Make a conclusion about the null hypothesis.
Our level of significance for this problem is 5%, so $\alpha = 0.05$. Since

$$ p\text{-value} = 0.0411 < 0.05 = \alpha, $$

we reject the null hypothesis.

We conclude that the population standard deviation score on the IQ test is not 15 points.

***

In [ ]:
#**VID=qWQ39iWrU-I**#

***

<small style="color:gray"><b>License:</b> This work is licensed under a [Creative Commons Attribution 4.0 International](https://creativecommons.org/licenses/by/4.0/) license.</small>

<small style="color:gray"><b>Author:</b> Taylor Baldwin, Mt. San Jacinto College</small>

<small style="color:gray"><b>Adapted From:</b> <i>Introductory Statistics</i>, by Barbara Illowsky and Susan Dean. Access for free at [https://openstax.org/books/introductory-statistics/pages/1-introduction](https://openstax.org/books/introductory-statistics/pages/1-introduction).</small>